apollo-router-benchmarks/Cargo.toml

```
@@ -1,6 +1,6 @@
 [package]
 name = "apollo-router-benchmarks"
-version = "0.11.0"
+version = "0.11.1"
```

I **think** we're trying to keep these all in lock-step at the moment, but I don't know if that strategy _actually_ makes a ton of sense or not.  To some degree, since these (are private / aren't published) we could just `0.0.0` all of them?  I don't think zeroing them out is a today thing, so maybe just making this `0.12.0` makes sense?

```suggestion
version = "0.12.0"
```

---

yeah that s a find/replace fail 😓